AccountGrain/AccountBusiness.cs

```diff
@@ -0,0 +1,89 @@
+﻿using CommunAxiom.Commons.Client.Contracts.Account;
+using Microsoft.Extensions.Configuration;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommunAxiom.Commons.Client.Grains.AccountGrain
+{
+    public class AccountBusiness

```

Comment:
I am wondering what is the difference between `Account` and `AccountBusiness`?

Reply:
Account business is the implementation of business logic applied when calling functionality in the grain. (think separation of concerns) The idea is that the grain is much like an api layer (like an aspnet controller) so it should contain communication and probably authorization level behavior, the Account Business holds the business rules associated with the functionality, we could put that in another project entirely, use it in another context (like unit testing) and it would still work. the AccountRepo simply holds the storage part of the grain. For now that's really simple but now that the grain storage is decoupled, we can hold much more information inside a single grain so we could find in the repo the typical data access features we find in a DAL object.